src/components/Common/ProgressBar.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import LoadingBar from 'react-redux-loading-bar';
+
+import './progress-bar.scss';
+
+export default class ProgressBar extends Component {
```

Comment:
Why are u using there functional component but in the TournamentIcon component vice versa?

Reply:
Good question, i will fix it